src/patterns/hide-the-page/index.md.njk

```diff
@@ -0,0 +1,92 @@
+---
+title: Hide the page
+description: Help users to hide the page quickly if they’re at risk of harm
+section: Patterns
+theme: Help users to…
+aliases:
+backlog_issue_id: 35
+layout: layout-pane.njk
+---
+
+This pattern gives the user a way to quickly hide the page from a potential perpetrator if they’re nearby.
```

Comment:
From @lfdebrux:
> Unsure about the phrase ‘potential perpetrator’... perpetrator of what?

I'd taken the language from domestic abuse groups, as it's pretty sensitive. But I appreciate it's not quite clear enough! How about:

```suggestion
This pattern gives the user a way to quickly hide the page from someone that's watching them and invading their privacy. For example, when someone is experiencing domestic abuse.
```